UPGRADE-6.3.md

```diff
@@ -155,6 +155,23 @@ done by adding the following line after the module list definition:
 There are additional helper methods to simplify adding typical backend modules, but it is optional to use these methods.
 See `src/CoreBundle/private/library/classes/pagedefFunctions.inc.php` for reference.
 
+## Main Menu Changes
+
+This release of Chameleon System features a new main menu that is displayed as a sidebar while the old main menu (now
+called "classic main menu" is deprecated and will be removed in a future release. The new menu was restructured and some
+menu items were renamed to improve comprehensibility of the menu structure.
+
+The content boxes of the classic main menu are unchanged to give users time to get accustomed to the new menu. It
+didn't make sense to keep old menu item names, though, so be aware that some menu items were changed. The changes
```

Comment:
"Though it didn't make sense to keep the old menu item names. So ...."

Reply:
Removed the comma before "though", now it should be easier to read. Your proposal divides related information into two sentences and so doesn't exactly hit what I intended to say.

Reply:
(That's the strength of English: Using short sentences. :-) )